src/storage/rdb_ziplist.h

```diff
@@ -41,4 +53,4 @@ class ZipList {
   Status peekOK(size_t n);
   void setPreEntryLen(uint32_t len) { pre_entry_len_ = len; }
   static uint32_t getEncodedLengthSize(uint32_t len);
-};
+};
```

Comment:
```suggestion
};

```

Keep the newline at the end of file.

Reply:
done